### PR TITLE
{vis}[goolf/1.7.20] KLayout v0.24.9

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar 

```
EasyBuildError: 'Irresolvable dependencies encountered: Qt/4.8.6-goolf-1.7.20'
```

File: easybuild/easyconfigs/k/KLayout/KLayout-0.24.9-goolf-1.7.20-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+unpack_options = '--strip-components=1'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('zlib', '1.2.8'),
+    ('Ruby', '2.3.3', '', True),
+    ('Qt', '4.8.6'),
+]
+
+install_cmd = './build.sh -qt $EBROOTQT -ruby $EBROOTRUBY/bin/ruby -python $EBROOTPYTHON/bin/python -bin %(installdir)s/bin -platform linux-64-gcc-release -option -j4'
```
Comment (boegel) [inline easybuild/easyconfigs/k/KLayout/KLayout-0.24.9-goolf-1.7.20-Python-2.7.11.eb]: @pescobar too long, `-j4` should not be hardcoded?

**Comment by boegel** [comment]
@pescobar ping?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**